train.py

```diff
@@ -64,6 +64,11 @@
   flags.DEFINE_integer(
       "batch_size", 1024,
       "How many examples to process per batch for training.")
+  
+  flags.DEFINE_integer(
+    "epochs", 100,
```

Comment:
I think it may make some sense to increase this to a much larger number, to mimic more closely the current behavior. 100 epochs may take a long time, but for some users, this may be too low (for example, if I start training today, and I expect to stop training at the end of the competition, I'd be annoyed to find out that training stopped early).